src/tools/ceph_monstore_tool.cc

```diff
@@ -655,6 +655,14 @@ static int update_pgmap_meta(MonitorDBStore& st)
     t->put(prefix, "full_ratio", bl);
   }
   {
+    auto backfillfull_ratio = g_ceph_context->_conf->mon_osd_backfillfull_ratio;
+    if (backfillfull_ratio > 1.0)
+      backfillfull_ratio /= 100.0;
```

Comment:
@dzafman but at https://github.com/ceph/ceph/pull/13968/files#diff-0a5db46a44ae9900e226289a810f10e8R165, i failed to find the logic to normalize it. https://github.com/ceph/ceph/pull/13968/files#diff-0a5db46a44ae9900e226289a810f10e8R1019 also.